lxd/instance/drivers/driver_lxc.go

```diff
@@ -5673,6 +5673,7 @@ func (d *lxc) Exec(req api.InstanceExecPost, stdin *os.File, stdout *os.File, st
 		return nil, fmt.Errorf("Failed to retrieve PID of executing child process")
 	}
 	d.logger.Debug("Retrieved PID of executing child process", log.Ctx{"attachedPid": attachedPid})
+	d.lifecycle(lifecycle.InstanceExec, log.Ctx{"command": strings.Join(req.Command, " ")})
```

Comment:
Does it look particularly ugly when not rendered as a string? If not, we should keep it as a list to avoid ambiguity around spaces in arguments.

Reply:
I don't know about ugly, but it writes each argument on a new line, something like:
```
command:
    - arg1
    - arg2
    - arg3
```
Given that it's exactly one command that's being executed, I figured a single string would more naturally represent what was received. Also, you could potentially parse the output to extract the command, or copy/paste it to somewhere else like this.

Reply:
Okay, then as a string we need to make sure the escaping is correct. Looks like https://pkg.go.dev/github.com/alessio/shellescape#QuoteCommand may do that for us.

Reply:
If the command is going to be parsable then keeping it in its current (yaml?) form as a list seems simple and the renderer could always convert it to a string later. Keeping it structured for as long as possible.

Reply:
Yeah, makes sense, so we should keep it as a `[]string` in the context and then if we want to do something more compact, the CLI can handle it and use something like shellescape there.